File: 2020/elixir2020/test/day01_test.exs
defmodule Elixir2020.Day01Test do
  use ExUnit.Case
  import Elixir2020.Day01

  test "part1" do
    assert part1("../inputs/01/input.txt") == 121_396
  end

  test "part2" do
    assert part2("../inputs/01/input.txt") == 73_616_634
  end
end

File: 2022/go2022/go_day01/day01_test.go
// Package main provides ...
package main

import "testing"

func TestPart1(t *testing.T) {
	t.Run("It calculates Part1", func(t *testing.T) {
		elves := Parse("../../inputs/01/input.txt")
		got := PartOne(elves)
		want := 69912
		if got != want {
			t.Errorf("got %q want %q", got, want)
		}
	})
}
func TestPart2(t *testing.T) {
	t.Run("It calculates Part2", func(t *testing.T) {
		elves := Parse("../../inputs/01/input.txt")
		got := PartTwo(elves)
		want := 208180
		if got != want {
			t.Errorf("got %q want %q", got, want)
		}
	})
}
